docs/api/update.md

```diff
@@ -0,0 +1,293 @@
+# Vald Update APIs
+
+## Overview
+
+Update Service is responsible for updating vectors which are already inserted in the `vald-agent` components.
+
+```bash
+service Update {
+    rpc Update(payload.v1.Update.Request) returns (payload.v1.Object.Location) {}
+
+    rpc StreamUpdate(stream payload.v1.Update.Request) returns (stream payload.v1.Object.Location) {}
+
+    rpc MultiUpdate(payload.v1.Update.MultiRequest) returns (stream payload.v1.Object.Locations) {}
+}
+```
+
+## Update RPC
+
+Update RPC is the method to update a single vector instead of a inserted vector.
```

Comment:
```suggestion
Update RPC is a method to update a single vector.
```

Reply:
https://github.com/vdaas/vald/blob/master/docs/api/insert.md?plain=1#L19
I'd like to use the same wording.

Reply:
I think the sentence of this part is incorrect. `update xxx instead of xxx`, so I suggested to remove that part.

Reply:
```suggestion
Update RPC is the method to update a single vector.
```
Is this ok?